src/rest_framework_dso/views.py

```diff
@@ -328,7 +328,7 @@ def _parse_accept_crs(self, value: Optional[str]) -> Optional[crs.CRS]:
 
         try:
             accept_crs = crs.CRS.from_string(value)
-        except ExternalValueError as e:
+        except (ExternalValueError, SyntaxError) as e:
```

Comment:
This is to fix a regression caused by a change in the errors that are  raised by django-gisserver

Reply:
Looks like you have the wrong version of django-gisserver installed.